modules/base.nix

```diff
@@ -216,9 +216,11 @@ in
     '' + (if (config.androidVersion >= 10) then ''
       echo "\$(call inherit-product-if-exists, robotnix/config/system.mk)" >> target/product/handheld_system.mk
       echo "\$(call inherit-product-if-exists, robotnix/config/product.mk)" >> target/product/handheld_product.mk
-    '' else ''
+    '' else if (config.androidVersion >= 8) /* FIXME Unclear if android 8 has these... */ then ''
```

Comment:
It is unclear whether something similar would be needed for Android 7.

It is unclear where Android 8 would fall here.

Reply:
This is where we hook into the existing build system to add additional packages / robotnix configs.  For now, we don't have to support robotnix module customizations with older versions of android.